src/com/sap/piper/DefaultValueCache.groovy

```diff
@@ -1,32 +1,79 @@
 package com.sap.piper
 
+import com.sap.piper.MapUtils
+
 import com.cloudbees.groovy.cps.NonCPS
 
 @API
 class DefaultValueCache implements Serializable {
     private static DefaultValueCache instance
 
+    private static String DEFAULT_PROJECT_CONFIG_FILE_NAME = '.pipeline/config.yml'
```

Comment:
I would like to avoid "file" as trailing part of name since it not a file (`java.io.File`). Would `DEFAULT_PROJECT_CONFIG_FILE_PATH` also be OK from your point of view?

Reply:
done